skimage/exposure/exposure.py

```diff
@@ -64,30 +115,27 @@ def histogram(image, nbins=256):
              "computed on the flattened image. You can instead "
              "apply this function to each color channel.")
 
+    if source_range is None:
+        warn('The default value of rescale will change to `dtype` in version 0.16')
```

Comment:
rescale -> source_range. I think it also helps to mention the full function name:

"The default value of source_range in skimage.exposure.histogram will change to 'dtype' in version 0.16"